### ISSUE TITLE
Automatic tagging of Pull Requests and Issues

### ISSUE THREAD

**Issue by czgdp1807** [sympy/sympy-bot#65]
I don't know if this is feasible, however, I would still like the sympy-bot to tag the PRs with labels, at least related to the module mentioned in the release notes. This would help the new contributors to help in improving sympy by finding the PRs and Issues of the module that interests them.

**Comment by asmeurer** [comment]
Adding a tag based on the release notes modules would be easy. Beyond that, how would it know what labels to add to an issue or pull request?

**Comment by czgdp1807** [comment]
One way can be to check which module's code is changed, may be, by checking the diff on github. Is that possible?

**Comment by asmeurer** [comment]
Yes, though that likely won't give more information than the release notes entry for most pull requests.

**Comment by czgdp1807** [comment]
Yes, but that will be definite as compared to release notes entries, which are sometimes wrong, especially with new contributors. 
Moreover, if possible, we can analyse the diff in a more detailed manner, like the affected class will give more information. I am unable to figure out how zulip-bot is able to tag PRs so intelligently. 
For the time being, we can go with module changes/release notes for labels.
